source/languages/en/riak/ops/building/installing/erlang.md

```diff
@@ -141,6 +142,21 @@ To install packages for graphics support, use this command:
 sudo apt-get install libwxbase2.8 libwxgtk2.8-dev libqt4-opengl-dev
 ```
 
+Next create a `bin` directory in your home directory.
+
+```bash
+mkdir ~/bin
+```
+
+Then run the following to update your `PATH`:
+
+```bash
+export JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64
+PATH=$JAVA_HOME/bin:$PATH
+PATH=~/bin:$PATH
+export PATH
+```
```

Comment:
Likewise here, let's just double-check with someone in Eng to make sure that this is a solid recommendation for every user (and not just something that's best for internal devs).